packages/react/src/radio-group/RadioGroup.tsx

```diff
@@ -21,25 +16,24 @@ const mapGapToOrientation = {
 } as const;
 
 export const RadioGroup = forwardRef<HTMLDivElement, RadioGroupProps>(
-  ({ children, flexDirection = "column", ...props }, ref) => {
+  (
+    { children, flexDirection = "column", orientation = "vertical", ...props },
+    ref,
+  ) => {
     const { restProps, sprinkleProps } = extractSprinkles(props);
 
     return (
       <Flex
         asChild
-        flexDirection={flexDirection}
+        flexDirection={orientation === "vertical" ? "column" : "row"}
         gap={mapResponsiveValue(
           flexDirection,
```

Comment:
we also need to address orientation here:
```suggestion
          flexDirection ?? orientation === "vertical" ? "column" : "row",
```

Reply:
addressed bhai